### PR TITLE
OKTA-550605 Token hook preview feature update

### PR DESCRIPTION
**Pull request by brianduffield-okta**
## Description:
- **What's changed?** Update to the following docs to add preview tab feature for the Token Inline Hook:
   * [Token inline hook guide](https://developer.okta.com/docs/guides/token-inline-hook/nodejs/main/#test-the-token-inline-hook)
   * [Common Hook set up steps](https://developer.okta.com/docs/guides/common-hook-set-up-steps/nodejs/main/#troubleshoot-hook-implementations)
   * [Inline hook concept](https://developer.okta.com/docs/concepts/inline-hooks/#troubleshooting)
   * [Hooks best practices](https://developer.okta.com/docs/reference/hooks-best-practices/)
- **Is this PR related to a Monolith release?** 2022.12.0

### Resolves:

* [OKTA-550605](https://oktainc.atlassian.net/browse/OKTA-550605)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/concepts/inline-hooks/index.md [hunk 1]
```diff
@@ -210,8 +210,10 @@ For more information on implementing inline hooks, see the documentation for spe
...
 
-The [Okta System Log](/docs/reference/api/system-log/) captures events related to inline hook setup and execution, which you can use to troubleshoot your implementation. To see descriptions of the relevant event types, query the Event Types catalog with the query parameter `inline_hook`:
+A hook preview feature is available for the following hooks: SAML, telephony, token, and registration inline hooks. Use this feature to view a sample request body and evaluate and troubleshoot the response from your external service. See [Troubleshoot hook implementations](/docs/guides/common-hook-set-up-steps/nodejs/main/#troubleshoot-hook-implementations) and [Preview an inline hook](https://help.okta.com/okta_help.htm?type=oie&id=ext-preview-inline-hooks).
 
-<https://developer.okta.com/docs/reference/api/event-types/?q=inline_hook>
+The [Okta System Log](/docs/reference/api/system-log/) also captures events related to inline hook setup and execution, which you can use to troubleshoot your implementation. To see descriptions of the relevant event types, query the Event Types catalog with the query parameter `inline_hook`:
+
+<https://developer.okta.com/docs/reference/api/event-types/>
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/concepts/inline-hooks/index.md:217]: Not sure if you're using this format for a reason? Or can we use the regular format:
```
See [Event Types inline hook query](https://developer.okta.com/docs/reference/api/event-types/?q=inline_hook)
or 
See [Event Types inline hook query](/docs/reference/api/event-types/?q=inline_hook)
```

File: packages/@okta/vuepress-site/docs/guides/token-inline-hook/main/index.md [hunk 2]
```diff
@@ -137,16 +137,40 @@ This extension renders the [ID token](/docs/reference/api/oidc/#id-token), and i
...
 
 * The Okta-Hosted-Login sample application (`samples-nodejs-express-4`) is ready to authenticate users from your Okta org.
 *  The external service (Glitch.com project) is ready with code to receive and respond to an Okta token inline hook call.
 *  The Okta org is set up to call the external service when a token inline hook is triggered by a user sign-in from the Okta-Hosted-Login sample application, and ready to receive a response.
 
 >**Note:** Make sure you have users assigned to your application and at least one user is part of the [Patients data store](/docs/guides/token-inline-hook/#check-against-the-data-store/) in your Glitch application.
 
+### Preview the token inline hook
+
+1. Navigate to Inline Hooks (**Workflow** > **Inline Hooks**) in your Admin Console.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/token-inline-hook/main/index.md:152]:
```suggestion
1. In the Admin Console, go to **Workflow** > **Inline Hooks**.
```
This is just a suggestion from infodev guidelines.

File: packages/@okta/vuepress-site/docs/guides/token-inline-hook/main/index.md [hunk 3]
```diff
@@ -137,16 +137,40 @@ This extension renders the [ID token](/docs/reference/api/oidc/#id-token), and i
...
+
+1. Click the token inline hook name (in this example, "Patient Token Hook").
+
+1. Click **Preview**.
+
+1. From the **Configure Inline Hook request** block, complete the following fields:
+
+    * **Select a user**: A user in your org associated with your application.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/token-inline-hook/main/index.md:160]:
```suggestion
    * **Select a user**: A user in your org associated with your application
```

File: packages/@okta/vuepress-site/docs/guides/token-inline-hook/main/index.md [hunk 4]
```diff
@@ -137,16 +137,40 @@ This extension renders the [ID token](/docs/reference/api/oidc/#id-token), and i
...
+    * **Select an application**: Your OIDC sample application name.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/token-inline-hook/main/index.md:161]:
```suggestion
    * **Select an application**: Your OIDC sample application name
```

File: packages/@okta/vuepress-site/docs/guides/token-inline-hook/main/index.md [hunk 5]
```diff
@@ -137,16 +137,40 @@ This extension renders the [ID token](/docs/reference/api/oidc/#id-token), and i
...
+    * **Select an authorization server**: Your authorization server name. In this example, `default`.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/token-inline-hook/main/index.md:162]:
```suggestion
    * **Select an authorization server**: Your authorization server name. In this example, use `default`.
```

File: packages/@okta/vuepress-site/docs/guides/token-inline-hook/main/index.md [hunk 6]
```diff
@@ -137,16 +137,40 @@ This extension renders the [ID token](/docs/reference/api/oidc/#id-token), and i
...
+    * **Select a grant type**: Your application's grant type. In this example, `Authorization Code`.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/token-inline-hook/main/index.md:163]:
```suggestion
    * **Select a grant type**: Your application's grant type. In this example, use `Authorization Code`.
```

File: packages/@okta/vuepress-site/docs/guides/token-inline-hook/main/index.md [hunk 7]
```diff
@@ -137,16 +137,40 @@ This extension renders the [ID token](/docs/reference/api/oidc/#id-token), and i
...
+    * **Select scopes**: The granted scopes.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/token-inline-hook/main/index.md:164]:
```suggestion
    * **Select scopes**: The granted scopes
```

File: packages/@okta/vuepress-site/docs/guides/token-inline-hook/main/index.md [hunk 8]
```diff
@@ -137,16 +137,40 @@ This extension renders the [ID token](/docs/reference/api/oidc/#id-token), and i
...
+
+    >**Note:** Based on your grant type selection, preview fields may vary.
+
+1. From the **Preview example Inline Hook request** block, click **Generate Request**. You should see the user's request information in JSON format that is sent to the external service.
+
+   >**Note:** You can also **Edit** this call for development or testing purposes.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/token-inline-hook/main/index.md:170]:
```suggestion
   >**Note:** You can also click **Edit** to modify this call for development or testing purposes.
```

**Review by vanngo-okta — APPROVED** [review]
A few style nits. Approving to not hold you back.

File: packages/@okta/vuepress-site/docs/concepts/inline-hooks/index.md (hunk 1 above)
Comment (brianduffield-okta) [reply packages/@okta/vuepress-site/docs/concepts/inline-hooks/index.md:217]: Historically left in there, I assumed, to highlight the query parameter. I don't think this direct link is working, though (it isn't after I checked), so, I'll remove this bit. I thought I fixed it, but I think that was for Event Types.

File: packages/@okta/vuepress-site/docs/guides/token-inline-hook/main/index.md (hunk 2 above)
Comment (brianduffield-okta) [reply packages/@okta/vuepress-site/docs/guides/token-inline-hook/main/index.md:152]: Thanks, yes, I used that for updates to the same content on infodev. I'm fine to use this style.

File: packages/@okta/vuepress-site/docs/concepts/inline-hooks/index.md (hunk 1 above)
Comment (brianduffield-okta) [reply packages/@okta/vuepress-site/docs/concepts/inline-hooks/index.md]: Updated. Thanks.

**Comment by brianduffield-okta** [comment]
merging to release branch

**Pull request merged**